cli/package.json

```diff
@@ -32,7 +32,7 @@
     "package.json"
   ],
   "engines": {
-    "node": ">=18.17.0"
+    "node": ">=20.18.0"
```

Comment:
is there any strict requirement to having this?

just merged #2009 to support versions without global crypto.

The CLI itself should, if possible, not be a blocker for people to run.